### PR TITLE
Added favicon to the analytics platform

### LINKED ISSUE #849: Add favicon on the Analytics platform
**Describe the solution you'd like**
We would like to add a favicon to the [Analytics platform](https://staging-analytics.airqo.net/) site that is the same as that used on the [AirQo website](https://airqo.africa/).

**Additional context**
- You can find the favicons in the [this website codebase folder](https://github.com/airqo-platform/AirQo-frontend/tree/master/website/frontend/assets)
- The [Analytics platform](https://staging-analytics.airqo.net/) codebase can be found [here](https://github.com/airqo-platform/AirQo-frontend/tree/master/platform)

### PR DESCRIPTION
**Pull request by patrikbodnar**
Succesfully added  favicon to the analytics platform. 👍 

#### Summary of Changes (What does this PR do?)
- Succesfully added  favicon to the analyrics platform for issue #849 that makes the issue done. Thank you for giving me this opportunity. 👍 
- Closes #849 

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):
- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### Screenshots (optional)
image

### CODE REVIEW

File: platform/src/pages/_app.jsx [hunk 1]
```diff
@@ -1,5 +1,13 @@
 import '@/styles/global.scss';
+import Head from "next/head";
 
 export default function App({ Component, pageProps }) {
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <Head>
+        <link rel="shortcut icon" href="icons/favicon.ico" />
```
Comment (Codebmk) [inline platform/src/pages/_app.jsx:8]: Thanks, @patrikbodnar . We have a custom Document(_document.jsx) with a Head component where you can add the link. Kindly move that there.

Comment (patrikbodnar) [reply platform/src/pages/_app.jsx:8]: Ok, I will do it right now :)

Comment (patrikbodnar) [reply platform/src/pages/_app.jsx]: @Codebmk , I have moved that there. :)

**Review by Codebmk — APPROVED** [review]
LGTM

File: platform/src/pages/_app.jsx (hunk 1 above)
Comment (Codebmk) [reply platform/src/pages/_app.jsx]: Great @patrikbodnar

**Review by 123MwanjeMike — APPROVED** [review]
Great work @patrikbodnar 
Looks good to me

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**